### DISCUSSION TITLE
Prior studies upfront

### DISCUSSION THREAD

**Discussion by Ravindra-Desai128** [nroduit/Weasis#461] [Q&A]
Hello, @nroduit. If a patient contains multiple studies, is it possible to list them under "ALL STUDIES"? Currently, I have to open each study within the same patient to have them listed under "ALL STUDIES." Is there a way to load the list of studies present for a single patient without having to open each one individually?

Thanks in advance

**Comment by nroduit** [comment]
Please specify the use case as I don't understand the request. See the [use of DICOM Explorer](https://weasis.org/en/tutorials/dicom-explorer).

**Comment by Ravindra-Desai128** [comment]
@nroduit 
![image](https://github.com/nroduit/Weasis/assets/141710987/eee3d359-06d7-442d-9dd8-6bf424c12f50)


For example,  same Patient contain  2 study , I have to open 2 times one by one rt?  instead of that if i open any one of the study remaining studies should list in the dropdown no need to open again

![image](https://github.com/nroduit/Weasis/assets/141710987/c9961201-44a0-4bc2-9704-349eea51d53d)
 patient contain 3 study if i open any one of the study rest should list in the studies list dropdown

**Comment by nroduit** [comment]
Weasis is not connected to an archive; it simply [receives the exams defined before launching the viewer](https://weasis.org/en/basics/customize/integration). Exam selection is done server-side via [the weasis-pacs-connector API ](https://github.com/nroduit/weasis-pacs-connector#launch-weasis)(also via the [standard IHE API to launch a viewer](https://github.com/nroduit/weasis-pacs-connector#launch-weasis-with-ihe-iid-profile)), or directly via Weasis [querying DICOMWeb services](https://weasis.org/en/basics/customize/integration/index.html#download-directly-with-dicomweb-restful-services) directly.

In the [Query/Retrieve section](https://weasis.org/en/tutorials/dicom-import/index.html#dicom-queryretrieve), you can select the examinations you want, as Weasis is making the request to the DICOM archive.

**Comment by Ravindra-Desai128** [comment]
ok thanks
